presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Enable spatial joins (broadcast and distributed) for ST_Equals, ST_Overlaps, ST_Crosses, and ST_Touches.
+* Fix a bug where index joins would fail with the error "driver should never block".  Now queries will wait until a configured timeout for the indexes to load.  The timeout can be configured with the configuration property ``index-loader-timeout`` and the session property ``index_loader_timeout``.
+* Move ``AggregationNode`` to SPI. Connectors can now push down aggregation to table scan.
+* Move ``ProjectNode`` to SPI. Connectors can now push down project to table scan.
+* Add ``task_partitioned_writer_count`` session property to allow to set number of concurrent writers for partitioned (bucketed) writes independently.
+* Removing syntax support for ``SET PATH``, ``CURRENT_PATH`` (Presto server do not use this information)
+* Fix NullPointerException in spatial join with geometry collections in the build side.
+* Add ``IPPREFIX`` type and :func:`ip_prefix` function.
+* Improve performance for queries with ``FULL OUTER JOIN`` where join keys have the :func:``COALESCE`` function applied.
+* Improve performance for Unnest Operator by using DictionaryBlocks.
+* Add :func:`differential_entropy` functions to compute differential entropy.
+* Rename ``Block#getObject`` to ``Block#getBlock`` and remove unnecessary ``clazz`` parameter.
+* Improve performance for PartitionedOutputOperator. It can be enabled by the ``optimized_repartitioning`` session property or the ``experimental.optimized-repartitioning`` configuration property.
+
+Hive Changes
+____________
+* Improve CPU load on coordinator by reducing the cost of serializing ``HiveSplit``.
+* Fix a bug that might lead to corruption when writing sorted table in recoverable grouped execution mode.
+* Fix ORC stripe skipping when using bloom filter.
+* Add subfield pruning to reading of Parquet files so that only necessary subfields are extracts from struct columns.
+* Add an option to switch file system on S3 to allow users to configure their own ones by setting the configuration property ``hive.s3-file-system-type=HADOOP_DEFAULT`` and providing file system dependency in ``core-site.xml``.
+* Reduce GC pressure from Parquet reader by constraining the maximum column read size.
```

Comment:
Is there a configuration property for this?

Reply:
@caithagoras thanks for the catch. There're both session property and config property for this. I changed the line to 
```
* Reduce GC pressure from Parquet reader by constraining the maximum column read size. It can be configured by the ``parquet_max_read_block_size`` session property or the ``hive.parquet.max-read-block-size`` configuration property.
```
cc @zhenxiao Can you confirm?

Reply:
Looks good to me

Reply:
Shall we add:
 Subfield pruning for Parquet: scan only necessary data for Parquet structs

Reply:
Hi @zhenxiao subfield pruning has been included in the following line under `Hive Changes`:
```
* Support sub-field pruning when reading Parquet files, so that only necessary the
  sub-fields are extracted from struct columns.
```

Reply:
nice. LGTM